Point_set_processing_3/include/CGAL/IO/read_ply_points.h

```diff
@@ -165,6 +165,11 @@ namespace internal {
       c = static_cast<unsigned char>(s);
     }
 
+    void read_ascii(std::istream& stream, double& d) const
+    {
+      stream >> iformat(d);
+    }
+
```

Comment:
This change is already present in the PR #2373, that is about to be merged in `master`, see:
https://github.com/CGAL/cgal/pull/2373/files#diff-aa78664827be8dc329359cf1b4cb0080R174

Unless you want to apply this PR to previous versions of CGAL, such as CGAL-4.11, I suggest we cancel this PR #2446.

Reply:
You can cancel it.